components/symbol/src/overrides/symbol.js

```diff
@@ -57,7 +58,7 @@ const symbolStyles = (_, { symbol, width, height, viewBoxWidth, viewBoxHeight })
 
 const blenderStyles = (_, { symbol, width, height, viewBoxWidth, viewBoxHeight }) => {
 	const props = { symbol, width, height, viewBoxWidth, viewBoxHeight };
-	const baseStyles = symbolStyles(_);
+	const baseStyles = symbolStyles(_, defaultProps);
```

Comment:
blenderStyles was not working, icon blenderStyles is using a similar implementation and defaultProps was passed in for this same line for icon, did the same here and it started working.